components/LyricsEditor.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
```

Comment:
For me, the editor didn't work very well for https://telegra.ph/Miracle-07-20

Reply:
Could you try again now?

Reply:
Works great now!